ansible-st2-local/roles/st2web/tasks/main.yml

```diff
@@ -1,26 +0,0 @@
----
-
-- name: Download webui
-  get_url: url=https://downloads.stackstorm.net/releases/st2/{{ st2web_version }}/webui/webui-{{ st2web_version }}.tar.gz dest=/tmp/
-
-- name: Create /opt/opt/stackstorm/static/
-  sudo: yes
-  file: state=directory path=/opt/stackstorm/static/
-
-- name: Uncompress webui
-  sudo: yes
-  unarchive: copy=no src=/tmp/webui-{{ st2web_version }}.tar.gz  dest=/opt/stackstorm/static/
-
-- name: Set config
-  sudo: yes
-  template: src=config.js.j2 dest=/opt/stackstorm/static/webui/config.js 
-
-- name: Change directory owner from default stanley to our local user
-  sudo: yes
-  file: path=/opt/stackstorm/static/webui owner={{ ansible_ssh_user }} group={{ ansible_ssh_user }} recurse=yes 
```

Comment:
Should this be `ansible_ssh_user` perhaps it should rather be root or `arteria_user`?